### DISCUSSION TITLE
『Qiitaでの「エンジニア」の定義』のリンクについて教えてくださいませ

### DISCUSSION THREAD

**Discussion by TORIFUKUKaiou** [increments/qiita-discussions#397] [ヘルプ] [answered]
いつもQiitaさんを利用させていただいております。ありがとうございます。感謝しかございません。ありがとうございます。

とても細かい点ですが質問をさせてください。

「エンジニア」へのリンクとして、[コミュニティガイドライン](https://help.qiita.com/ja/articles/qiita-community-guideline)内のリンクを使わせていただきたいとおもうときがあります。

右横のもくじみたいなところから、『Qiitaでの「エンジニア」の定義』を押してみますと、Chrome バージョン: 108.0.5359.124（Official Build） （x86_64）の場合、『行動指針 ‐ Qiitaの目指す世界』へジャンプします。

これは、『行動指針 ‐ Qiitaの目指す世界』と『Qiitaでの「エンジニア」の定義』の両方とも、 `https://help.qiita.com/ja/articles/qiita-community-guideline#qiita` になっているからだとおもいます。

『Qiitaでの「エンジニア」の定義』に直接遷移できる方法がございましたらご教示いただけますとありがたいです。

ご確認のほどよろしくお願いします。

**Comment by tanaka-asami415** [comment]
@TORIFUKUKaiou 
こちらこそいつもありがとうございます✨

ご不便をおかけして申し訳ありません。
リンクが間違っているため、年明け以降になってしまいますが修正いたします🙇
修正後には直接遷移できるようになるはずですので、しばらくお待ちいただけますと幸いです🙏

**Comment by TORIFUKUKaiou** [comment]
@tanaka-asami415 san

ご回答ありがとうございます。承知いたしました。

「私[待つわ](https://www.youtube.com/watch?v=OIf-KTBi_-8)」なんちゃって。

よろしくお願いします。

**Comment by wataru86** [comment, marked as answer]
@TORIFUKUKaiou

お待たせしました :bow:

使用しているライブラリの都合により影響範囲が大きく、目次のリンク自体はまだ修正ができていませんが、暫定対応としてそれぞれの箇所に飛べるリンクを用意しました。

- 「行動指針 ‐ Qiitaの目指す世界」
  - https://help.qiita.com/ja/articles/qiita-community-guideline#qiita
- 「Qiitaでの「エンジニア」の定義」
  - https://help.qiita.com/ja/articles/qiita-community-guideline#engineer
- 「Qiitaを利用するにあたってのマナー」
  - https://help.qiita.com/ja/articles/qiita-community-guideline#manner

目次のリンクの修正は、他の開発優先度なども踏まえて修正を検討します :bow:
ご不便をおかけして申し訳ございません。

**Reply by TORIFUKUKaiou** [reply]
ご連絡ありがとうございます！
